bin/cron/deploy_to_levelbuilder

```diff
@@ -16,32 +18,27 @@ MESSAGE_DTL_SKIPPED_NOTHING = 'robo-DTL is being skipped (nothing new to deploy)
 MESSAGE_DTL_SKIPPED_TOPIC = 'robo-DTL is being skipped (developers DTL was not yes). ' \
   'Please contact the DOTD for a manual deploy, if desired.'
 
-# Runs the given command on the given server.
-# @param [String] server The server (e.g., 'test') to run the command on.
-# @param [String] command The command to be run.
-# TODO(asher): Move this to a location for better reuse, deduplicating existing copies of it.
-def run_on(server, command)
-  # Double-escape the command string, once for each remote shell we're passing through.
-  system "ssh -t gateway.code.org ssh -t #{server} #{command.inspect.inspect}"
-end
-
 # Creates a new branch with the given name from the given commit on the test server
 # (without checking it out) and pushes it to GitHub.
 # @param [String] branch_name The name of the branch to create.
 # @param [String] commit The sha of the commit to base the branch at.
 # TODO(asher): Move this to a location for better reuse, deduplicating existing copies of it.
```

Comment:
Good Q. Definitely still relevant, though a slightly moving target as the `deploy_to_test`, `deploy_to_levelbuilder`, and `scoop_content` (soon to be `commit_content`) scripts change. Ideally (IMO), I'll resolve the TODO after the scripts settle, that way I know the best API to offer.

For example, I would have done things "wrong" originally, based on wjordan's (good) suggestion that the only cross environment interaction should be through PR merges.